test/data/bittorrent-dht-client.js

```diff
@@ -1,56 +1,56 @@
 module.exports = DHT
 module.exports.dgram = require('dgram') // allow override for chrome apps (chrome-dgram)
 
-var addrToIPPort = require('addr-to-ip-port')
-var bencode = require('bencode')
-var bufferEqual = require('buffer-equal')
-var compact2string = require('compact2string')
-var debug = require('debug')('bittorrent-dht')
-var dns = require('dns')
-var EventEmitter = require('events').EventEmitter
-var hat = require('hat')
-var inherits = require('inherits')
-var isIP = require('is-ip')
-var KBucket = require('k-bucket')
-var networkAddress = require('network-address')
-var once = require('once')
-var os = require('os')
-var parallel = require('run-parallel')
-var publicAddress = require('./lib/public-address')
-var sha1 = require('simple-sha1')
-var string2compact = require('string2compact')
-
-var BOOTSTRAP_NODES = [
+const addrToIPPort = require('addr-to-ip-port')
+const bencode = require('bencode')
+const bufferEqual = require('buffer-equal')
+const compact2string = require('compact2string')
+const debug = require('debug')('bittorrent-dht')
+const dns = require('dns')
+const EventEmitter = require('events').EventEmitter
+const hat = require('hat')
+const inherits = require('inherits')
+const isIP = require('is-ip')
+const KBucket = require('k-bucket')
+const networkAddress = require('network-address')
+const once = require('once')
+const os = require('os')
+const parallel = require('run-parallel')
+const publicAddress = require('./lib/public-address')
+const sha1 = require('simple-sha1')
+const string2compact = require('string2compact')
```

Comment:
This is a minimum set of edits on third party code (the example code linked to from the StandardJS website). We don't currently have a rule enforcing import over require so this passes linting. There is a plugin that can do this (https://github.com/benmosher/eslint-plugin-import) via `no-commonjs` and `no-amd` but I think that would be a separate PR from enforcing let/const.